src/System.Private.Xml/src/System/Xml/Serialization/ReflectionXmlSerializationWriter.cs

```diff
@@ -1400,7 +1400,20 @@ public static MemberInfo GetMember(Type declaringType, string memberName)
             MemberInfo[] memberInfos = declaringType.GetMember(memberName);
             if (memberInfos == null || memberInfos.Length == 0)
             {
-                throw new InvalidOperationException(SR.Format(SR.XmlInternalErrorDetails, $"Could not find member named {memberName} of type {declaringType.ToString()}"));
+                bool readFromBaseType = false;
+                if(declaringType.BaseType != null)
+                {
+                    memberInfos = declaringType.BaseType.GetMember(memberName);
```

Comment:
If the BaseType's base type has such member, does it work?

Can you please add some tests for the fix?